pkgs/tools/package-management/lix/doc/default.nix

```diff
@@ -1,14 +1,18 @@
 {
+  lib,
   src,
   rustPlatform,
   version,
   cargoHash ? null,
   cargoLock ? null,
 }:
-
+let
```

Comment:
all change of this file should disappear and this file should only be kept for <2.92 compat